inst/include/DoubleVariable.h

```diff
@@ -71,12 +73,9 @@ struct DoubleVariable : public Variable {
         const double a, const double b
     ) const {
         
-        int result{0};
-        for(size_t it = 0; it < values.size(); it++) {
-            if( !(values[it] < a) && !(b < values[it]) ) {
-                result += 1;
-            }
-        }
+        int result = std::count_if(values.begin(), values.end(), [&](const double v) -> bool {
```

Comment:
Can we use an unsigned integer for counts?

```suggestion
        size_t result = std::count_if(values.begin(), values.end(), [&](const double v) -> bool {
```

it's, again, more explicit about what we expect from that line